exporters/elasticsearch-exporter/README.md

```diff
@@ -125,7 +125,7 @@ Here is a complete, default configuration example:
       className: io.zeebe.exporter.ElasticsearchExporter
 
       args:
-        url: http://localhost:9200
+        urls: http://elastic-1:9200,http://elastic-2:9200
```

Comment:
If we use comma separated string, why not reuse the same parameter? This will make it backwards compatible, since no commas just mean it's a single URL.

Also here let's keep using the default values, so we should rather put something like:

```yaml
      args:
        # A comma separated list of URLs pointing to the Elasticsearch instances you wish to export to. 
        # For example, if you want to connect to multiple nodes for redundancy:
        # url: http://localhost:9200,http://localhost:9201
        url: http://localhost:9200
```